kore/src/Kore/Exec.hs

```diff
@@ -275,26 +279,34 @@ proveWithRepl definitionModule specModule = do
         axioms
         claims
 
-makeClaim :: (Attribute.Axiom, Rewrite) -> Claim Object
+assertSomeClaims :: Monad m => [claim] -> m ()
+assertSomeClaims claims =
+    Monad.when (null claims) . error
+        $   "Unexpected empty set of claims.\n"
+        ++  "Possible explanation: the frontend and the backend don't agree "
+        ++  "on the representation of claims."
```

Comment:
Would this happen because of using the wrong frontend and backend versions together?

Reply:
Yes. What I wanted to achieve was a quick way to fail the build (instead of returning #True for verifying the empty list of claims) as it happened before and also was happening in this PR